models/product-type/src/product-type-draft/transformers.ts

```diff
@@ -0,0 +1,21 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TProductTypeDraft, TProductTypeDraftGraphql } from '../types';
+
+const transformers = {
+  default: Transformer<TProductTypeDraft, TProductTypeDraft>('default', {
+    buildFields: ['attributes', 'attributeDefinitions'],
+  }),
+  rest: Transformer<TProductTypeDraft, TProductTypeDraft>('rest', {
+    buildFields: ['attributes'],
+    removeFields: ['attributeDefinitions'],
```

Comment:
I like this but it does mean the consumers will have to define each of them depending on their `build` use case. My care level is fairly low, but it's relevant and sets a precedent